src/mapper/pkg/metricexporter/otel_edge_metric.go

```diff
@@ -0,0 +1,93 @@
+package metricexporter
+
+import (
+	"context"
+	"time"
+
+	"github.com/otterize/network-mapper/src/mapper/pkg/config"
+	sharedconfig "github.com/otterize/network-mapper/src/shared/config"
+	"github.com/spf13/viper"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetricgrpc"
+	"go.opentelemetry.io/otel/exporters/stdout/stdoutmetric"
+	"go.opentelemetry.io/otel/metric"
+	sdk "go.opentelemetry.io/otel/sdk/metric"
+	"go.opentelemetry.io/otel/sdk/resource"
+	semconv "go.opentelemetry.io/otel/semconv/v1.21.0"
+)
+
+type OtelEdgeMetric struct {
+	meterProvider *sdk.MeterProvider
```

Comment:
Is there a reason we hold on to the MeterProvider here?

If there is, we should use the API `meter.MeterProvider`